editor/src/core/shared/import/project-health-check/utopia-requirements-service.ts

```diff
@@ -33,12 +36,29 @@ export function updateProjectRequirementsStatus(
 export function resetRequirementsResolutions(dispatch: EditorDispatch) {
   let projectRequirements = emptyProjectRequirements()
   updateProjectRequirementsStatus(dispatch, projectRequirements)
+}
+
+export function startPreParseValidation(dispatch: EditorDispatch) {
+  const checks = getRequirementsToCheck().preParse
   notifyOperationStarted(dispatch, {
-    type: 'checkRequirements',
-    children: Object.keys(projectRequirements).map((key) =>
-      importCheckRequirementAndFix(
+    type: 'checkRequirementsPreParse',
+    children: Object.keys(checks).map((key) =>
+      importCheckRequirementAndFixPreParse(
         key as ProjectRequirement,
-        initialTexts[key as ProjectRequirement],
+        checks[key as PreParseValidationRequirement].initialText,
```

Comment:
I would be very hesitant to use these casts because they absolutely will break things in weird ways if the value of `checks` changes.